test/index.spec.js

```diff
@@ -306,6 +277,29 @@ describe('hermes', function () {
       connection.emit('error', new Error('Some connection error'));
     });
 
+    describe('#persistent option', function () {
+      beforeEach(function (done) {
+        connectFinish();
+        channel.origSendToQueue = channel.sendToQueue;
+        channel.sendToQueue = sinon.spy(function (queueName, data, opts) {
+          expect(opts.persistent).to.equal(true);
+        });
+        done();
+      });
+
+      afterEach(function (done) {
+        channel.sendToQueue = channel.origSendToQueue;
+        delete channel.origSendToQueue;
+        done();
+      });
+
+      it('should send messages with true persistent opt', function (done) {
+        //connectFinish();
+        hermes.publish(TEST_QUEUE, {foo: 'bar'});
+        done();
+      });
```

Comment:
this actually just isn't testing anything... it's being set in the `beforeEach`, which is weird. could just spy on the function and then assert that it was called...